events/python/receive_kombu.py

```diff
@@ -0,0 +1,35 @@
+from kombu import Consumer, Exchange, Queue, Message
+
+from setup_kombu import (
+    ARTICLE_EXCHANGE_NAME,
+    DASHBOARD_QUEUE_NAME,
+    get_connection
+)
+
+
+def message_handler(body: str, message: Message) -> None:
+    print(f'[x] Received: {body}')
+    message.ack()
```

Comment:
is this necessary given the `no_ack=False` later on?

Reply:
> I found through later testing that if I did not manually ack in the handler then the messages were labelled as received but unacked by the server.
Same deal here in practice.